Jenkinsfile

```diff
@@ -39,14 +39,16 @@ pipeline {
           fi
           sudo sysctl -w net.ipv6.conf.all.disable_ipv6=1
           sudo sysctl -w net.ipv6.conf.default.disable_ipv6=1
+          sudo yum install -y python38-devel
```

Comment:
Cannot you add this to your image used for testing instead install it each pipeline run? I will approve it, but will be better if you have it as part of your image. WDYT?

Reply:
We use some very generic cloud images and I don't know how easy it will be to modify those. I'd prefer to keep this install here for now.

Reply:
That would require us to maintain a tool that keeps those custom images up-to-date.

What do you recommend that we use for that?

Reply:
That sounds a bit out of scope to me and more like something that could be taken up as it's own issue. If engineering is okay with adding a bit of time to their pipeline (I don't think python38-devel takes an insane amount of time to yum install) then I think we should go ahead and merge this to unblock QE on dropping python 3.6 support. Just my two cents :)

Reply:
I approved this PR and just thought that engineering are using some custom image in PSI where they installed already some other dependencies. If that's not the case I don't mind to keep it this way.